src/wagtail_factories/blocks.py

```diff
@@ -239,3 +245,26 @@ def _build(cls, model_class, document):
     @classmethod
     def _create(cls, model_class, document):
         return document
+
+
+# Only define ImageBlockFactory if Wagtail 6.3 or higher is installed
+if WAGTAIL_63_OR_HIGHER:
+    class ImageBlockFactory(StructBlockFactory):
+        image = factory.SubFactory(ImageChooserBlockFactory)
+        decorative = factory.Faker("boolean")
+        alt_text = factory.Sequence(lambda n: f"Alt text {n}")
+
+        class Meta:
+            model = ImageBlock
+
+        @classmethod
+        def _construct_struct_value(cls, block_class, params):
+            if image := params.get("image"):
+                decorative = params.get("decorative")
```

Comment:
Question: will `decorative` be in `params` if not supplied by the user? If not, should we default to `False` rather than `None`, for consistency?

Reply:
Yes, it will be a boolean value generated by Faker.